clash_lib/src/proxy/converters/shadowsocks.rs

```diff
@@ -46,6 +46,15 @@ impl TryFrom<&OutboundShadowsocks> for AnyOutboundHandler {
                         .try_into()
                         .map(OBFSOption::V2Ray)
                         .ok(),
+                    "shadow-tls" => s
```

Comment:
mind adding a docker test for this?

Reply:
Alright, but need some modification of docker test. Since the shadow-tls is a plugin of shadowsocks, we need to extend the docker test's ability by enabling it to start multiple images altogether

Reply:
or we shall create a new docker images, including the binary of shadowsocks&shadow-tls. what do you think

Reply:
i've extended the test utility and run multiple images at the same time as the preparation for the docker test. you can have a quick look, shall i merge it in this PR or in a separate one?

Reply:
yeah i think it's better to separate for easier review

Reply:
all right